### PR TITLE
upcoming: Update `useAccountAvailabilitiesQuery` query to use `getAll`

### PR DESCRIPTION
**Pull request by coliu-akamai**
## Description 📝
- Forgot to do this in the original PR 😅 , but basically just making our existing query for `useAccountAvailabilitiesQuery` use `getAll`

### Verification steps 
(How to verify changes)
- Verify that data still looks as expected when using this query and that pagination is gone 

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/queries/accountAvailability.ts [hunk 1]
```diff
@@ -3,30 +3,24 @@ import {
...
-    () => getAccountAvailabilities(params, filter),
-    {
-      enabled,
-      keepPreviousData: true,
-    }
+export const useAccountAvailabilitiesQuery = (enabled: boolean = true) =>
+  useQuery<AccountAvailability[], APIError[]>(
+    queryKey,
+    getAllAccountAvailabilitiesRequest,
+    { enabled, keepPreviousData: true }
```
Comment (abailly-akamai) [inline packages/manager/src/queries/accountAvailability.ts:17]: Can we keep the paginated query and add a new one? or give useAccountAvailabilitiesQuery the option to do either?

Comment (coliu-akamai) [reply packages/manager/src/queries/accountAvailability.ts:39]: Added new query! (v open to name changes 😅 )

Comment (bnussman-akamai) [reply packages/manager/src/queries/accountAvailability.ts:39]: A new query makes sense. We do that for most entities

Comment (abailly-akamai) [reply packages/manager/src/queries/accountAvailability.ts:39]: cool! ty - name is fine by me @coliu-akamai

**Review by abailly-akamai — APPROVED** [review]

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**